### PR TITLE
Doc add timeseries example

### PR DESCRIPTION
**Pull request by Iglesys347**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] Does `$ tox` pass with this change (including linting)?
- [ ] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Is there an example added to the examples folder (if applicable)?
- [ ] Was the change added to CHANGES file?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change

Adding a notebook with examples of Redis timeseries.

This PR relates to : #1744

### CODE REVIEW

File: docs/examples/timeseries_examples.ipynb [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+    "ts.mget([\"label1=1\"], with_labels=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Detention period\n",
+    "\n",
+    "You can specify a detention period when creating a timseries or when adding sample in a timeseries. Once the detention period is elapsed, the sample is removed from the timeseries."
```
Comment (chayim) [inline docs/examples/timeseries_examples.ipynb:359]:
```
    "You can specify a retention period when creating timseries objects or when adding a sample timeseries object. Once the retention period has elapsed, the sample is removed from the timeseries."
```

File: docs/examples/timeseries_examples.ipynb [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Timeseries\n",
+    "\n",
+    "`redis-py` supports [RedisTimeSeries](https://github.com/RedisTimeSeries/RedisTimeSeries/) which is a time-series-database module for Redis.\n",
+    "\n",
+    "This example shows quickly how to handle timeseries with `redis-py`."
```
Comment (chayim) [inline docs/examples/timeseries_examples.ipynb:11]:
```
    "This example shows how to handle timeseries data with `redis-py`."
```

**Comment by chayim** [comment]
Thanks @Iglesys347 for is awesome example! Tiny changes - thoughts? @dvora-h if you have perms feel free to merge it into this PR and then merge.

**Comment by Iglesys347** [comment]
Hi @chayim do I need to do anything on my side ?

**Comment by dvora-h** [comment]
@Iglesys347 Just if you can change it according to @chayim comments

File: docs/examples/timeseries_examples.ipynb [hunk 3]
```diff
@@ -0,0 +1,631 @@
...
+   ],
+   "source": [
+    "ts.mget([\"label1=1\"], with_labels=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Detention period\n",
```
Comment (dvora-h) [inline docs/examples/timeseries_examples.ipynb:357]: @Iglesys347 Another typo - Retention period

**Review by dvora-h — APPROVED** [review]

**Pull request merged**